### ISSUE TITLE
Feature Request: Network proxy setting

### ISSUE THREAD

**Issue by Jerry23011** [buresdv/Cork#112]
Since Cork doesn't follow the system proxy setting, a configurable proxy setting for Cork would be nice.
  
Users can set the proxy server they want to use, like the command in the terminal:
```
export http_proxy=http://127.0.0.1:1234/
export https_proxy=http://127.0.0.1:1234/
```

**Comment by buresdv** [comment]
Hmm, I would prefer if Cork would follow the system setting. I have to look into that.

**Comment by Jerry23011** [comment]
It would be even better if Cork could just follow the system setting. :)

**Comment by buresdv** [comment]
I agree, although I have no idea what I'm doing wrong 😅

**Comment by Jerry23011** [comment]
I'm wondering how Cork communicates with the terminal

If you open up a new terminal session, it will not use the system proxy, you'd have to set it up yourself through the method I mentioned above

That's why I was suggesting a manual proxy configuration through command

**Comment by buresdv** [comment]
The communication is nothing fancy, the app issues commands to the shell. I think I'll end up implementing something like your suggestion above in that case.

**Comment by buresdv** [comment]
Implemented in 1.1
